### DISCUSSION TITLE
Strange formatting in search results

### DISCUSSION THREAD

**Discussion by DaHunch** [scambier/obsidian-omnisearch#247] [General]
Hi - sorry, new to Github, hope this is ok to ask here.
Just wondering why my results come with this lack of spacing to the text?
Hurting my eyes - but maybe there's a reason?

Thanks for any tips/advice/condolences...

![image](https://github.com/scambier/obsidian-omnisearch/assets/135022979/d04ead4b-7c2f-4883-a7ba-62fe26c7f298)

**Comment by scambier** [comment]
This is a bug that should be fixed in the latest version (1.14.2). https://github.com/scambier/obsidian-omnisearch/issues/244

**Comment by DaHunch** [comment]
Ah - updated and all good!  Many thanks - that's great news.
